example-bots/rust/src/main.rs

```diff
@@ -34,12 +34,12 @@ struct State {
 }
 
 fn compute_move(state: State) -> std::string::String {
-    let my_player = if "player1" == &*state.you {
+    let my_player = if "player1" == state.you.as_ref() {
```

Comment:
From the rust docs it looks like as_ref is a generic way to do what you did. Makes sense from an idiomatic point of view, but it seems it is basically the same thing
```
#[stable(since = "1.5.0", feature = "smart_ptr_as_ref")]
impl<T: ?Sized, A: Allocator> AsRef<T> for Box<T, A> {
    fn as_ref(&self) -> &T {
        &**self
    }
}
```